### PR TITLE
update dep: simple8b @ b421ab40

### PR DESCRIPTION
**Pull request by benbjohnson**
Update the `simple8b` depedency to the latest version.

/cc @jwilder 

###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by benbjohnson** [comment]
@jwilder I left the `simple8b.Decode()` removal out so it'd be separate from the update.

File: tsdb/engine/tsm1/timestamp_test.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func Test_TimeEncoder(t *testing.T) {
 		t.Fatalf("Wrong encoding used: expected rle, got %v", got)
 	}
 
-	dec := NewTimeDecoder()
+	var dec TimeDecoder
```
Comment (jwilder) [inline tsdb/engine/tsm1/timestamp_test.go]: Doesn't this still need to be pointer?  `Init` mutates the fields.

Comment (benbjohnson) [inline tsdb/engine/tsm1/timestamp_test.go]: Go will use the pointer if the function's receiver is a pointer. So it automatically converts:

```go
dec.Init(b)
```

to

```go
(&dec).Init(b)
```

Comment (jwilder) [inline tsdb/engine/tsm1/timestamp_test.go]: Cool

**Comment by jwilder** [comment]
👍

**Pull request merged**